static/js/protein/chart.ts

```diff
@@ -1021,6 +1028,11 @@ export function drawVarGeneAssocChart(
     .attr("cy", (d) => y(d.id))
     .attr("r", "6")
     .style("fill", (d) => ERROR_BAR_VAR_COLOR[d.name])
+    // variant redirect
+    .on("click", function (d) {
+      console.log(d.associationID);
+      window.location.href = `${GRAPH_EXPLORER_REDIRECT}${d.associationID}`;
```

Comment:
we usually want to use window.open(<url>) which opens the url in a new tab or window.open(<url>, "_self") to open in current tab

Reply:
Done!